app/controllers/apo_controller.rb

```diff
@@ -36,7 +36,7 @@ def registration_options
       end
 
       ["#{name.truncate(60, separator: /\s/)} (#{col_id.delete_prefix('druid:')})", col_id]
-    end.sort_by(&:first) # before returning the list, sort by collection name
+    end.compact.sort_by(&:first) # before returning the list, sort by collection name
```

Comment:
Can you add a comment here about why this compact is necessary?  Otherwise someone in the future is likely to factor this out as they will see it as un-necessary.